console/server/src/main/java/quarks/console/server/ServerUtil.java

```diff
@@ -19,10 +19,18 @@ Licensed to the Apache Software Foundation (ASF) under one
 
 package quarks.console.server;
 
+import java.io.IOException;
 import java.io.File;
 import java.io.FilenameFilter;
 import java.net.URL;
+import java.nio.file.Files;
```

Comment:
pls remove above FilenameFilter import as it's now no longer used and causes compile warning in Eclipse.